filebeat/module/mysql/slowlog/test/mariadb-10.1.21.log

```diff
@@ -0,0 +1,7 @@
+# Time: 180613 11:04:36
+# User@Host: root[root] @ localhost [121.0.0.1]
+# Thread_id: 5  Schema:   QC_hit: No
+# Query_time: 2.000652  Lock_time: 0.000000  Rows_sent: 1  Rows_examined: 0
+SET timestamp=1528898676;
+select sleep(2);
+
```

Comment:
Should this empty line be here? Would also be nice to have at least 2 log entries in the example log file to make sure also multiline works as expected.